### PR TITLE
project-detail-page

### LINKED ISSUE #134: [FEATURE] Create Project's detail page [FE]
### Description

Create projects details page.

- Detail page contains more info of the project
- User will navigate to `project details page` when he/she clicks on contribute button

### PR DESCRIPTION
**Pull request by AdityaPainuli**
Closes #134 

### You can check it out on /projects/{any-number}
- write now it's not protected or attached route and everything is just hardcoded as of now 

### Feature I proposed - 
1. Adding a comment component to comment on a particular post 
2. Ask the user to post an image of a certain ratio to preview on the details page to make it look good. 

### Quick screenshots of the UI - 

image

image

### CODE REVIEW

File: pages/projects/[projectId].tsx [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
+            </span>
+            <span className="group flex w-max cursor-pointer flex-wrap items-center space-x-1 rounded-full bg-background-2 px-4 py-2 text-[15px] text-blue-500">
+              Mongo
+            </span>
+          </div>
+        </div>
+        <button className="m-auto flex w-max items-center space-x-4 rounded-md bg-orange-500 p-2 px-4 py-2 text-xl font-semibold text-white hover:bg-orange-400">
+          <span>Contribute now</span> <BsBoxArrowUpRight />
+        </button>
+      </div>
```
Comment (rohitdasu) [inline pages/projects/[projectId].tsx:76]: please don't hardcode, we already have the API. 
also please integrate with the flow (example on clicking of `contribute` button it should redirect to the details page)

**Review by rohitdasu — CHANGES_REQUESTED** [review]

**Comment by rohitdasu** [comment]
Alignment issue ⚠️ 
Screenshot 2022-10-05 at 10 29 54 PM

**Comment by leewan-09** [comment]
I think the **Skill required** is not the appropriate word to use here

**Comment by AdityaPainuli** [comment]
> I think the **Skill required** is not the appropriate word to use here

Yes, I am changing it with the tech stack. any better heading suggestions ??

**Pull request closed without merging**